ios/Mappers.swift

```diff
@@ -658,12 +658,19 @@ class Mappers {
     }
 
     class func mapFromPaymentMethod(_ paymentMethod: PaymentMethod) -> NSDictionary {
+        var cardPresentMapped: NSDictionary?
+        if let cardPresent = paymentMethod.cardPresent{
+            cardPresentMapped = mapFromCardPresent(cardPresent)
+        }
+        var interacPresentMapped: NSDictionary?
+        if let interacPresent = paymentMethod.interacPresent{
+            interacPresentMapped = mapFromCardPresent(interacPresent)
+        }
+
         let result: NSDictionary = [
-            "id": paymentMethod.stripeId,
-            "created": convertDateToUnixTimestamp(date: paymentMethod.created) ?? NSNull(),
-            "customer": paymentMethod.customer ?? NSNull(),
```

Comment:
were these intentionally removed? 🤔

Reply:
did you mean to rm `type` then and leave in `customer`?  looks like customer is there on both and metadata might be a good one to add while here too

> there is already a paymentMethodId field top level

the native SDKs duplicate this so may be worth it for consistency? I could see it being valuable if someone wanted to only pass around the PaymentMethod piece to work with 🤷‍♂️